app/services/decorators/v5/disposable_income_result_decorator.rb

```diff
@@ -37,13 +38,13 @@ def basic_attributes
       attr_reader :summary, :gross_income_summary
 
       def net_employment_income
-        gross_income_summary.gross_employment_income + summary.employment_income_deductions + summary.fixed_employment_allowance
+        @employment_income_subtotals.gross_employment_income.to_f + summary.employment_income_deductions + summary.fixed_employment_allowance
```

Comment:
I don't think these to_f calls are needed any more - they were an result of using large-precision BigDecimals (causing JSON strings "1.34452344" vs 1.34452344) , which have been fixed universally now

Reply:
The `to_f` is also fixing situations where values may be nil but we're doing addition with them. So it's shorthanding `(value || 0)`.

Reply:
sorry missed that - this is a sum! I think see above about null objects - if we need a zero rather than nil, maybe we can have an entity that does the transformation once rather than making all (even if only 1) clients have to cope.

Reply:
I've managed to remove some of the `to_f`s, but in the process found another reason why some of them have to be in the decorators: Sometimes, a value is `0`, e.g. when we get it by doing `some_empty.collection.sum(:value)` But the ispec tests will fail if the value is not expressed as `0.0`. `0` is a 'correct' representation of the value, it's only the v5 API design that promises to represent this value with a decimal point, so I think the v5 decorator is the right place to do the transformation when it's needed.